main.go

```diff
@@ -87,6 +99,25 @@ func main() {
 		os.Exit(1)
 	}
 
+	var provisionerFactory provisioner.Factory
+	if runInTestMode {
+		provisionerFactory = fixture.New
```

Comment:
We used to log prominently when we have the test/demo provisioners selected, that probably doesn't hurt.

Reply:
Fixed